src/styles/main.css

```diff
@@ -66,16 +74,54 @@ li > a:hover {
   color: hsl(0, 100%, 30%);
 }
 
+.cards-display-section {
+  margin-bottom: 8rem;
+}
+
+.movie-item {
+  background-color: hsl(0, 100%, 30%);
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  padding: 10px;
+}
+
+.movie-title {
+  margin: 5px 0;
+  color: hsl(0, 100%, 30%);
+}
+
+.btn {
+  color: hsl(0, 100%, 30%);
+  border: none;
+  background-color: #000;
+  padding: 10px;
+  width: 100%;
+}
+
 footer {
   display: flex;
-  justify-content: flex-end;
+  justify-content: center;
   align-items: center;
   padding-right: 2rem;
   padding-left: 2rem;
-  height: 50px;
+  height: 100px;
   background-color: #000;
   color: #fff;
   position: fixed;
   bottom: 0;
```

Comment:
when user scroll down the footer is still there. It would be better user experience and allow more space for the item to show if you change the footer position, 
<img width="912" alt="image" src="https://github.com/Zohra-Neda/JavaScript-Capstone-Project/assets/118947090/bf6e948a-a67e-4b44-90a8-bb79c9e57412">